recipes/homes.rb

```diff
@@ -24,7 +24,7 @@
 end
 
 nfs_export '/data/homes/' do
-  network (1..26).map { |octet| '10.1.1.' + octet.to_s + '/32' }
+  network (1..254).map { |octet| '10.162.136.' + octet.to_s + '/24' }
```

Comment:
I think you mean `/32` otherwise that would be 254 `/24` subnets? Also can we reduce that to how many workstations there are?